dbtmetabase/metabase.py

```diff
@@ -723,7 +724,10 @@ def increase_indent(self, flow=False, indentless=False):
                         creator = self.api("get", f"/api/user/{exposure['creator_id']}")
                     except requests.exceptions.HTTPError as error:
                         creator = {}
-                        if error.response.status_code != 404:
+                        if (
+                            error.response is not None
+                            and error.response.status_code != 404
+                        ):
```

Comment:
this was updated due to mypy complaining

Reply:
We probably want to raise if it's none too, so:

```py
if not error.response or error.response.status_code != 404:
  raise
```